src/main/java/org/apache/sysml/api/python/SystemML.py

```diff
@@ -59,6 +376,40 @@ def __init__(self, sc, *args):
             self.ml = sc._jvm.org.apache.sysml.api.MLContext(sc._jsc, monitorPerformance, setForcedSparkExecType)
         except Py4JError:
             traceback.print_exc()
+            
+    def start(self):
+        try:
+            prevGlobalMLContext = MLContext.globalMLContext
+        except AttributeError:
+            pass
```

Comment:
It would be cleaner to set `globalMLContext = None` and `prevGlobalMLContext = None` as proper class variables at the top of the class so that there is never an `AttributeError`.